CHANGELOG.md

```diff
@@ -30,6 +30,7 @@
 - `[jest-snapshot]` Improve colors when snapshots are updatable ([#9132](https://github.com/facebook/jest/pull/9132))
 - `[jest-snapshot]` Ignore indentation for most serialized objects ([#9203](https://github.com/facebook/jest/pull/9203))
 - `[jest-transform]` Create `createTranspilingRequire` function for easy transpiling modules ([#9194](https://github.com/facebook/jest/pull/9194))
+- `[jest-transform]` [**BREAKING**] Do not automatically wrap functions in a module wrapper ([#9253](https://github.com/facebook/jest/pull/9253))
```

Comment:
Um, we still wrap the functions, but don't inject any globals, right? Maybe we can phrase it better

Reply:
Yeah, good call